R/set_llm.R

```diff
@@ -0,0 +1,22 @@
+#' export
+set_llm <- function(gitai, 
+                    provider = "openai",
+                    model = "gpt-4o-mini") {
+  
+  if (provider == "openai") {
+
+    gitai$llm <- elmer::chat_openai(
+      model = model,
+      echo = "none"
+    )
+  }
+
+  invisible(gitai)
+}
+
+#' export
+set_system_prompt <- function(gitai, system_prompt) {
+
+  gitai$llm$system_prompt <- system_prompt
```

Comment:
There is already method for that in `GitAI` - could be something like:

```suggestion
  gitai$set_prompt(system_prompt)
```

Reply:
What is more, for the time being if we run `set_system_prompt()` just after `initialize_project()` without `set_llm()` (as I did by mistake when testing manually) there is no error returned.

Reply:
Ok. I've changed how setting prompts works. 
Now we would use set_prompt() function that sets system (but can set also part of user prompt in the future if needed). And it uses system_prompt which is now active binding (for setting and getting the prompt). 
And it rises error if there is no llm set.